### DISCUSSION TITLE
Episode 205: 🇳🇱 C++ Under the Sea Live 🇳🇱 Phil Nash & Jonathan Müller

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#104] [Episode]
Episode Link: https://adspthepodcast.com/2024/10/25/Episode-205.html

Feel free to write comments or ask questions 🙂

**Comment by Brian-M-J** [comment]
Does C++ Under the Sea not have a YouTube channel? How would one watch the recorded talks? Do they even record the talks?

**Reply by codereport** [reply]
They recorded the talks but haven't set up a youtube channel from the looks of it. I was told at the conference they do plan on uploading them and starting a channel so it is just a matter of when. I will try to remember to mention it on the podcast when I see the channel appear :+1:

**Reply by AndreasSchaetti** [reply]
The channel is online now, although there have been only three talks uploaded so far:
https://www.youtube.com/@cppunderthesea

**Reply by codereport** [reply]
Thanks! I will announce on the next podcast recording!
